### PR TITLE
Use catkin tools

### PR DESCRIPTION
**Pull request by k-okada**
testing at http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/jsk.rosbuild/

### CODE REVIEW

File: jsk.rosbuild [hunk 1]
```diff
@@ -14,8 +14,8 @@ function usage {
 
 function error {
     echo "export ROS_HOME=$ROS_HOME"
-    echo "source $ROS_INSTALLDIR/setup.bash"
-    (which rosrun && rosrun rosunit clean_junit_xml.py; echo "done")
+    echo "source $ROS_INSTALLDIR/devel/setup.bash"
+    (cd $ROS_INSTALLDIR; catkin_test_results --all; echo "done")
```
Comment (garaemon) [inline jsk.rosbuild]: argument is required on indigo

**Comment by garaemon** [comment]
whith current jsk.rosinstall, no euslisp is installed from source. is it correct?

File: jsk.rosbuild (hunk 1 above)
Comment (k-okada) [inline jsk.rosbuild]: once you run catkin catkin, CATKIN_TEST_RESULTS_DIR is set and you do not have to set arguments

Comment (garaemon) [inline jsk.rosbuild]: I see

**Comment by garaemon** [comment]
LGTM

**Comment by garaemon** [comment]
By these workarounds, I think almost of the packages can be built with catkin build.

* https://github.com/tork-a/rwt_ros/pull/16
* https://github.com/jsk-ros-pkg/jsk_recognition/pull/663
* https://github.com/start-jsk/rtmros_common/pull/637
* https://github.com/start-jsk/rtmros_gazebo/pull/169
* https://github.com/start-jsk/rtmros_gazebo/pull/170
* https://github.com/start-jsk/rtmros_tutorials/pull/163
* https://github.com/start-jsk/rtmros_tutorials/pull/164
* https://github.com/jsk-ros-pkg/jsk_demos/pull/141

**Comment by garaemon** [comment]
* https://github.com/tork-a/rtmros_nextage/pull/139
* https://github.com/start-jsk/rtmros_gazebo/pull/172

By merging these PRs, jsk.rosbuild works with catkin tools

**Pull request merged**